src/cameras/sidebar.jsx

```diff
@@ -1,7 +1,8 @@
+import React from 'react'
 import * as Material from '@material-ui/core'
-import { Header } from 'cameras/header'
+
 import { Ids } from 'cameras/list'
-import React from 'react'
+import { Header } from 'cameras/header'
 
 export const Sidebar = () => <Material.Grid item xs={12} md={3}>
```

Comment:
Let's extract this component and all sidebar related files to a `/sidebar` directory.

Reply:
Done!

Reply:
Let's also move all the sidebar related files here. What I'm referring to are the components used inside the `<Sidebar/>` one (`<Header/>` and `<Ids/>`)

Reply:
Done!